rules-reviewed/eap6/java-ee/seam/seam-ui.windup.xml

```diff
@@ -17,26 +17,60 @@
+                    <link href="https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/migration_guide/#migrate-eap5-component-upgrade-reference" 
+                          title="JBoss EAP 5 Component Upgrade Reference"/>
                     <tag>seam</tag>
                     <tag>jsf</tag>
                 </classification>
             </perform>
         </rule>
-        <rule id="seam-ui-jsf-00001">
+        <rule id="seam-ui-jsf-00002">
             <when>
-                <xmlfile matches="//s:button">
-                    <namespace prefix="s" uri="http://jboss.com/products/seam/taglib" />
-                    <namespace prefix="seam" uri="http://jboss.org/schema/seam/taglib" />
+                <xmlfile matches="//*[namespace-uri()='http://jboss.org/schema/seam/taglib']">
                 </xmlfile>
             </when>
             <perform>
+                <classification issue-display-mode="all" title="Seam 2.3 Tag library" effort="1" category-id="mandatory">
+                    <description>JSF template includes Seam 2.3 Tag library reference, review the usage of Seam UI controls and try to replace them with suggestions for concrete Seam UI control.
+                        For EAP 7 you need to migrate all Seam UI tags to JSF 2.2 UI tags.
+                        Seam 2.3 was supported only on JBoss EAP 6 and JBoss EAP 7, which now includes JSF 2.2 and Hibernate 5, does not support Seam 2.3 due to end of life of Red Hat JBoss Web Framework Kit. 
+                        It is recommended that you rewrite your Seam components using Weld CDI beans and replace Seam UI tags with JSF 2.2 UI tags.
+                        Start with removing tag library reference.
+                    </description>
+                    <link href="https://access.redhat.com/solutions/84873" title="Why Seam is not included in JBoss EAP 6"/>
+                    <link href="https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/migration_guide/#migrate-eap5-component-upgrade-reference" 
+                          title="JBoss EAP 5 Component Upgrade Reference"/>
+                    <tag>seam</tag>
+                    <tag>jsf</tag>
+                </classification>
+            </perform>
+        </rule>
+        <rule id="seam-ui-jsf-00001">
+            <when>
+                <or>
+                    <xmlfile matches="//s:button">
+                        <namespace prefix="s" uri="http://jboss.com/products/seam/taglib" />
+                    </xmlfile>
+                    <xmlfile matches="//s:button">
+                        <namespace prefix="s" uri="http://jboss.org/schema/seam/taglib" />
+                    </xmlfile>
```

Comment:
This duplication of xmlfile in or element is due I was not able to have one xmlfile with multiple namespaces to work on both cases.